tests/models/test_hooks.py

```diff
@@ -374,6 +414,7 @@ def test_trainer_model_hook_system_fit(tmpdir):
         'on_fit_end',
         'teardown',
     ]
+    called = [c['name'] for c in called]
```

Comment:
Note to reviewers: these tests will be updated in future PRs

Reply:
if the PR already somewhere?

Reply:
yes: https://github.com/PyTorchLightning/pytorch-lightning/pull/7738